app/src/main/java/com/github/dodobest/upbitapi/UpbitViewHolder.kt

```diff
@@ -5,10 +5,15 @@ import com.github.dodobest.upbitapi.databinding.CoinItemBinding
 import com.github.dodobest.upbitapi.model.UpbitTickerDataWithKoreanName
 
 class UpbitViewHolder(
-    private val binding: CoinItemBinding,
-) : RecyclerView.ViewHolder(binding.root) {
+    private var _binding: CoinItemBinding?,
+) : RecyclerView.ViewHolder(_binding?.root ?: throw IllegalArgumentException()) {
+    private val binding get() = _binding!!
```

Comment:
```suggestion
    binding: CoinItemBinding?,
) : RecyclerView.ViewHolder(_binding?.root ?: throw IllegalArgumentException()) {
    private var _binding = binding
```
생성자에는 언더바(_)를 쓰는 것은 지양하는 것이 좋아요.

Reply:
private 데이터에 언더바를 붙이고, public 데이터에 언더바 없이 선언하는 것이 컨벤션이라고 생각했습니다.

[c82001f](https://github.com/DoTheBestMayB/UpbitAPI/pull/35/commits/c82001ffaf088209ab935fb9027528ce8ca34be5)

Reply:
private setter public getter인 경우에만, private 변수에 언더바를 붙이는 것이 컨벤션입니다.

Reply:
클래스 생성자에 있는 view binding은 null처리를 해주지 않아도 되나요?
suggested change로 적어주신 코드에서 생성자에 있는 binding은 var을 붙이지 않아서 null 처리를 해줄 수가 없기 때문에
_binding 변수를 null처리해도 binding 변수가 view binding을 참조하고 있기 때문에 문제가 될 것 같습니다!

`private val _binding get() = binding ?: throw IllegalArgumentException()`와 같이 선언하면, _binding은 view binding을 참조하는 것이 아니라 단순히 getter 역할만 하기 때문에 생성자로 입력받은 변수만 null처리하면 될 것 같습니다.

[06f91cb](https://github.com/DoTheBestMayB/UpbitAPI/pull/35/commits/06f91cb97729b7a9972e4e39ff865116721c1108)